lib/arjdbc/informix/adapter.rb

```diff
@@ -45,20 +45,6 @@ def jdbc_column_class
       ::ActiveRecord::ConnectionAdapters::InformixColumn
     end
 
-    module ColumnMethods
```

Comment:
in theory, this must be getting included somewhere ... so maybe keep the remove or find the place

Reply:
Good point. It's used on line 33, but it seems we can remove that entire function, since it's optional. Shall I indeed remove the reference there and the `self.column_selector` function?